ps2-decompile/utils.h

```diff
@@ -0,0 +1,14 @@
+#ifndef UTILS_H
+#define UTILS_H
+
+#include "callbacks.h"
+#include "shared.h"
+#include <stdio.h>
+#include <usbd.h>
```

Comment:
```suggestion
#include <stdio.h>

#include <usbd.h>

#include "callbacks.h"
#include "shared.h"
```

I want header order to be standard, 3rd party, own. They need a space between so clang-format will leave them.